src/order/rel_classes.lean

```diff
@@ -227,6 +227,10 @@ noncomputable def is_well_order.linear_order (r : α → α → Prop) [is_well_o
   linear_order α :=
 by { letI := λ x y, classical.dec (¬r x y), exact linear_order_of_STO' r }
 
+/-- Derive a `has_well_founded` instance from a `is_well_order` instance. -/
+def is_well_order.has_well_founded [has_lt α] [hwo : is_well_order α (<)] :
```

Comment:
This could be used elsewhere in `mathlib`, so someone should open up a PR for this once this is merged.

Reply:
Where could it be used and how can we find those places?

Reply:
It could be used on `enat` after #14416, on `cardinal` after #13410, and on `(ordinal.out o).α`.

Reply:
I've just looked at this again. Is it really worth replacing for instance
```
instance : has_well_founded enat := ⟨(<), lt_wf⟩
```
by
```
instance : has_well_founded enat := is_well_order.to_has_well_founded 
```
?